src/test/java/com/jprotractor/integration/CommonFunctions.java

```diff
@@ -0,0 +1,108 @@
+package com.jprotractor.integration;
+
+import org.apache.commons.lang.exception.ExceptionUtils;
+
+import java.net.BindException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.io.File;
+import java.io.IOException;
+// import java.nio.file.Files;
+// import java.nio.file.Paths;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.firefox.internal.ProfilesIni;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.phantomjs.PhantomJSDriver;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import com.jprotractor.NgBy;
+import com.jprotractor.NgWebDriver;
+import com.jprotractor.NgWebElement;
+
+public class CommonFunctions {
+	private static WebDriver seleniumDriver;
+	static WebDriverWait wait;
+	static int implicitWait = 10;
+	static int flexibleWait = 5;
+	static long pollingInterval = 500;
+	static int width = 600;
+	static int height = 400;
+	// set to true for Desktop, false for headless browser testing
+	static boolean isDestopTesting = true;
+	static boolean isCIBuild =  false;
+
+	public static WebDriver getSeleniumDriver() throws IOException {
+		checkEnvironment();
+		if (isDestopTesting ){
+			// For desktop browser testing, run a Selenium node and Selenium hub on port 4444	
+			DesiredCapabilities capabilities = new DesiredCapabilities("firefox", "", Platform.ANY);
+			FirefoxProfile profile = new ProfilesIni().getProfile("default");
+			profile.setEnableNativeEvents(false);
+			capabilities.setCapability("firefox_profile", profile);
+			seleniumDriver = new RemoteWebDriver(new URL("http://127.0.0.1:4444/wd/hub"), capabilities);
+			return seleniumDriver;
+		} else {
+			seleniumDriver = new PhantomJSDriver();
+		}
+		return seleniumDriver;
+	}
+
+	public static boolean checkEnvironment() {
+		Map env = System.getenv();
+
+		if (env.containsKey("TRAVIS") && env.get("TRAVIS").equals("true")) {
+			isCIBuild =  true;
+			isDestopTesting = false;
+		}
+		return isCIBuild;
+	}
+
+	protected static String getScriptContent(String filename) {
```

Comment:
as far as I see, you are using this function for html files, so I think it should be renamed.